tests/backup/backup_share_test.go

```diff
@@ -2162,230 +2162,186 @@ var _ = Describe("{SharedBackupDelete}", func() {
+				log.InfoD("Recent backup name [%s] ", recentBackupName)
+				_, err = backupSuccessCheck(recentBackupName, orgID, 0, 0, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying the success of recent backup named [%s]", recentBackupName))
 			}
-			log.InfoD("All Accesses are toggled and operations are performed")
-
 		})
-
 	})
+
 	JustAfterEach(func() {
 		defer EndPxBackupTorpedoTest(contexts)
 		ctx, err := backup.GetAdminCtxFromSecret()
-		log.FailOnError(err, "Fetching px-central-admin ctx")
-
-		//Get scheduleUid
-		log.Infof("Get scheduleUid")
-		scheduleUid, err := GetScheduleUID(backupName, orgID, ctx)
-		log.FailOnError(err, "Error in fetching schedule UID for %v", backupName)
-		log.InfoD("scheduleUid - %v", scheduleUid)
-
-		//Delete Schedule Backup-
-		log.Infof("Delete Schedule Backup-")
-		err = DeleteSchedule(backupName, scheduleUid, periodicPolicyName, schPolicyUid, orgID)
-		log.FailOnError(err, "Error deleting Schedule backup %v", backupName)
-
-		//GetAll backups -
+		dash.VerifySafely(err, nil, "Fetching px-central-admin ctx")
+		scheduleUid, err := GetScheduleUID(scheduleName, orgID, ctx)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Fetching uid of schedule named [%s]", scheduleName))
+		allScheduleBackupNames, err := Inst().Backup.GetAllScheduleBackupNames(ctx, scheduleName, orgID)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Fetching all schedule backup names of schedule named [%s]", scheduleName))
+		err = DeleteSchedule(scheduleName, scheduleUid, periodicSchedulePolicyName, periodicSchedulePolicyUid, orgID)
+		dash.VerifySafely(err, nil, fmt.Sprintf("Verifying deletion of schedule named [%s] along with its backups %v and schedule policies [%v]", scheduleName, allScheduleBackupNames, []string{periodicSchedulePolicyName}))
 		backupNames, err := GetAllBackupsAdmin()
-		log.FailOnError(err, "Fetching admin backups")
-
-		//Delete Backup
-		backupDriver := Inst().Backup
+		dash.VerifySafely(err, nil, "Fetching admin backups")
 		for _, backupName := range backupNames {
-			log.InfoD("Deleting backup - %v", backupName)
-			backupUID, err := backupDriver.GetBackupUID(ctx, backupName, orgID)
-			log.FailOnError(err, "Failed while trying to get backup UID for - %s", backupName)
-			log.Infof("About to delete backup - %s", backupName)
-			backupDeleteResponse, err := DeleteBackup(backupName, backupUID, orgID, ctx)
-			log.FailOnError(err, "Backup [%s] could not be deleted with delete response %s", backupName, backupDeleteResponse)
+			backupUID, err := Inst().Backup.GetBackupUID(ctx, backupName, orgID)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Fetching the uid of the backup named [%s]", backupName))
+			log.InfoD("About to delete backup - %s", backupName)
+			resp, err := DeleteBackup(backupName, backupUID, orgID, ctx)
+			log.InfoD("DeleteBackup response for backup named [%s] - %v", backupName, resp)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Verifying the deletion of the backup named [%s] in admin ctx", backupName))
```

Comment:
We can now remove these steps of deleting backups since it is being handled in `CleanupCloudSettingsAndClusters`

Reply:
Can this be done in a separate PR for refactoring test cases?

Reply:
Done.